CommonMark breaks travis

Travis is broken with below error message. Probably because of a newer version of `commonmark`

```
$ python tools/check.py .
Traceback (most recent call last):
  File "tools/check.py", line 841, in <module>
    main(parsed_args)
  File "tools/check.py", line 825, in main
    res = validate_single(fn, template=template)
  File "tools/check.py", line 708, in validate_single
    validate_file = validator(filepath)
  File "tools/check.py", line 66, in __init__
    ast = self._parse_markdown(self.markdown)
  File "tools/check.py", line 73, in _parse_markdown
    parser = CommonMark.DocParser()
AttributeError: 'module' object has no attribute 'DocParser'
make: *** [check] Error 1
```

Fix: Revert to an older version?

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-CommonMark
+CommonMark==0.5.4
 pandocfilters
 PyYAML
 pep8

